recipes/emotion/index.ts

```diff
@@ -124,7 +107,7 @@ export default RecipeBuilder()
     explanation: `Update the Babel configuration to use Emotion's plugin and preset to enable some advanced features.`,
     singleFileSearch: paths.babelConfig(),
     transform(program: Collection<j.Program>) {
-      addBabelPlugin(program)
+      addBabelPlugin(program, "@emotion")
```

Comment:
Oops, I totally missed this in  #2156. Glad to see it'll be fixed here. 👍

Reply:
@wKovacs64 I saw your PR right after I submitted mine!